DOCKER-network/dockernet

```diff
@@ -0,0 +1,264 @@
+    # remove the remaining containers
+    os.system('docker container kill $(docker ps -q)')
+    os.system('docker system prune -f')
+
+
+def run(arguments):
+    container = arguments['container']
+    cmd = arguments['cmd']
+    docker_cmd = 'docker exec -it ' + container + ' bash -c \"cd /opt/data/devnet && ' + cmd + '\"'
+    os.system(docker_cmd)
+
+
+if __name__ == '__main__':
+    # initialize commands
+    cmd_list = {'init': init, 'start': start, 'update': update, 'stop': stop, 'run': run}
+
+    # Initialize Parser
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(dest='subparsers', help='sub-command help')
+
+    # Initialize init Parser
+    parser_init = subparsers.add_parser('init', help='generate docker compose files')
+    parser_init.add_argument('--validators', default='4', help='number of validator nodes')
+    parser_init.add_argument('--nonvalidators', default='1', help='number of non validator nodes')
+    parser_init.add_argument('--loglevel', default='4', help='level of logging for each node')
+    parser_init.add_argument('--total_funds', default='1000000000', help='initial funds for the network')
+    parser_init.add_argument('--maturity_time', default='4', help='staking maturity time in blocks')
+    parser_init.add_argument('--deleg_rewards_maturity_time', default='4', help='delegation rewards maturity in blocks')
+    parser_init.add_argument('--rewards_interval', default='1', help='number of reward blocks stored in one record')
+    parser_init.add_argument('--funding_deadline', default='10', help='governance proposal funding deadline in blocks')
+    parser_init.add_argument('--voting_deadline', default='12', help='governance proposal voting deadline in blocks')
+    parser_init.add_argument('--timeout_commit', default='1000', help='time between each block creation in (ms)')
+    parser_init.add_argument('--docker', action='store_true', default=True, help='docker flag used by olfullnode init')
+    parser_init.add_argument('--subnet', default='\\\"10.5.0.0/16\\\"', help='subnet of all node IP addresses')
+    parser_init.add_argument('--empty_blocks', action="store_true", help='create empty blocks')
+    parser_init.add_argument('--default', action='store_true', help='use default docker compose files')
+    parser_init.add_argument('--image', default='olprotocol', help='name of docker image')
+    parser_init.add_argument('--protocol_path', default='~/go/protocol', help='protocol source code path')
+    parser_init.add_argument('--docker_path', default='~/oldata/docker', help='docker shared volume for nodes')
+    parser_init.add_argument('--docker_bin', default='~/oldata/docker/bin', help='path for protocol executables')
+
+    # Initialize start Parser
+    parser_start = subparsers.add_parser('start', help='start network with generated docker compose files')
+    parser_start.add_argument('--docker_path', default='/home/kevin/oldata/docker', help='docker shared volume for nodes')
```

Comment:
I think you should remove machine specific paths

Reply:
yeah maybe we can set the default path to smth like `$OLDATA/docker`